include/pros/motors.hpp

```diff
@@ -1020,6 +1020,25 @@ class Motor_Group {
 	/** These functions let programmers configure the behavior of motor groups **/
 	/****************************************************************************/
 
+	/**
+	 * Indexes Motor in the Motor_Group in the same way as an array.
+	 * 
+	 * \param i
+	 *        The index value in the motor group.
+	 *
+	 * \return the appropriate Motor reference, return first motor and set errno on error.
+	 */ 
+	pros::Motor& operator[](int i);
+
+
+	/**
+	 * Indexes Motor in the Motor_Group in the same way as an array.
+	 * 
+	 *
+	 * \return the size of the vector containing motors
+	 */ 
+	std::int32_t size();
+
```

Comment:
Please add proper ERRNO documentation and other relevant information as seen in other comments.